website/common/locales/en/generic.json

```diff
@@ -197,5 +197,7 @@
     "online": "online",
     "onlineCount": "<%= count %> online",
     "loading": "Loading...",
-    "userIdRequired": "User ID is required"
+    "userIdRequired": "User ID is required",
```

Comment:
quite sure we have this message somewhere else and anyway it should be an ApiMessage not a locale string